conans/client/remote_manager.py

```diff
@@ -129,6 +130,9 @@ def get_package(self, pref, dest_folder, remote, output, recorder):
         t1 = time.time()
         try:
             pref = self._resolve_latest_pref(pref, remote)
+            snapshot = self._call_remote(remote, "get_package_snapshot", pref)
+            if not is_package_snapshot_complete(snapshot):
+                raise NotFoundException("")
```

Comment:
Please, say something like "Package pref='xxx' not found"...

Reply:
The exception is caught down to raise the real exception, but I will add a message